pkg/tfvars/powervs/powervs.go

```diff
@@ -0,0 +1,57 @@
+// Package powervs contains Power Virtual Servers-specific Terraform-variable logic.
+package powervs
+
+import (
+	"encoding/json"
+
+	"github.com/openshift/cluster-api-provider-powervs/pkg/apis/powervsprovider/v1alpha1"
+)
+
+type IBMCloud struct {
+	IBMCloudAPIKey      string `json:"ibmcloud_api_key"`
+	IBMCloudRegion      string `json:"ibmcloud_region"`
+	IBMCloudZone        string `json:"ibmcloud_zone"`
+	IAASClassicUserName string `json:"iaas_classic_username"`
+	IAASClassicAPIKey   string `json:"iaas_classic_api_key"`
+}
```

Comment:
these json values show up in $INSTALL_DIR/terraform.powervs.auto.tfvars.json. It seems to be convention that they all start with the provider name, so can you prepend `powervs` to them for consistency?

Reply:
e.g. `powervs_zone` `powervs_iaas_classic_username`

Reply:
unless you think `powervs_ibmcloud_zone` is better. (thoughts @mkumatag)?

Reply:
n/m. going with `powervs_zone`, after i discovered that the power-iaas ones aren't 1-1 mappings for ibmcloud ones.